{{cookiecutter.project_slug}}/{{cookiecutter.project_module}}/app.py

```diff
@@ -34,12 +34,14 @@
 )
 
 
-def init_app(application: Flask, interface: Api):
+def init_app(application, interface):
     if os.environ["ENVIRONMENT"] == "production":
         application.config.from_object(settings.Production)
     elif os.environ["ENVIRONMENT"] == "testing":
         application.config.from_object(settings.Testing)
     else:
         application.config.from_object(settings.Development)
+    logging.basicConfig(level=application.config['LOGLEVEL'],
```

Comment:
I like this although we may want to consider http://flask.pocoo.org/docs/dev/logging/.

Reply:
Ok, should definitely have looked over that first. I'll add updates later. I think it's important to try to get this right from the start in order to arrive at a point where we are actively using the loghouse stack to get insights into application logs and not just metrics.